docker/firewheel.dockerfile

```
@@ -0,0 +1,105 @@
+FROM ghcr.io/sandia-minimega/minimega/minimega:master AS minimega
```

```suggestion
FROM ghcr.io/sandia-minimega/minimega:master AS minimega
```

This path was updated as of https://github.com/sandia-minimega/minimega/pull/1552